dashboard/lib/tasks/asset_sync.rake

```diff
@@ -14,9 +14,21 @@ namespace :assets do
     app.assets = Sprockets::Railtie.build_environment(app)
     Sprockets::Railtie.build_manifest(app).compile('application.js')
   end
+
+  desc 'Copy digested assets to non-digested file paths'
+  task no_digests: :environment do
+    assets = Dir.glob("#{dashboard_dir}/public/assets/js/**/*")
+    regex = /-\w{32}(?:\.\w+$)/
+    assets.each do |file|
+      next if File.directory?(file) || file !~ regex
+      non_digested = file.sub(regex, '')
+      FileUtils.cp(file, non_digested)
+    end
+  end
 end
```

Comment:
If a newcomer to the codebase finds this PR's description then they'll be fine.  It's more if a newcomer comes across this function and wants to know its input & output at a glance.  An example of what an input asset and its matching output asset looks like could make it quicker to understand how the function works versus figuring out what each of the various steps do.